src/dwg.c

```diff
@@ -3192,3 +3205,230 @@ dwg_log_dataflags (const int _loglevel, const int maxlevel,
         HANDLER (OUTPUT, "  vert_alignment (128)\n");
     }
 }
+
+static int
+Dwg_Object_Ref_compare (const Dwg_Object_Ref *key, const Dwg_Object_Ref **ppR)
```

Comment:
static methods don't need the `Dwg_` prefix. `ref_cmp` is enough.

Reply:
OK. Thanks.